CBLAS/src/CMakeLists.txt

```diff
@@ -120,5 +120,9 @@ set_target_properties(
   VERSION ${LAPACK_VERSION}
   SOVERSION ${LAPACK_MAJOR_VERSION}
   )
+target_include_directories(cblas PUBLIC
```

Comment:
CMake minimum version is set to `2.8.10`:
* https://github.com/Reference-LAPACK/lapack/blob/e774b4339ebea509161ec9a75656662f64cd177e/CMakeLists.txt#L1

Command `target_include_directories` is not available:
* https://cmake.org/cmake/help/v2.8.10/cmake.html#section_Commands

This can be fixed by updating minimum requirement to 2.8.12:
* https://cmake.org/cmake/help/v2.8.12/cmake.html#command:target_include_directories

If update to `2.8.12` is acceptable, alternatively you can use `INCLUDES DESTINATION include` here:
* https://github.com/Reference-LAPACK/lapack/blob/e774b4339ebea509161ec9a75656662f64cd177e/CMakeLists.txt#L102-L107

Docs:
* https://cmake.org/cmake/help/v2.8.12/cmake.html#command:install